app/serializers/recipe_serializer.rb

```diff
@@ -1,8 +1,35 @@
 class RecipeSerializer
-  incliude FastJsonapi::ObjectSerializer
-  attributes :ingredients
+  def self.call(recipe)
```

Comment:
This overall isn't too bad.... I mean, I didin't have to hand roll it, but I was expecting a HUGE hash here!  Nice work getting it done!